package.json

```diff
@@ -26,6 +26,8 @@
         "build": "tsc -p .",
         "clean": "rimraf dist/* test-results/*",
         "test": "jest",
+        "tslint": "tslint -c ./tslint.json -p ./tsconfig.json",
```

Comment:
this is covered by #7 already (I don't think a separate fix script is necessary, I think adding `-- --fix` is reasonable enough)

Reply:
I will remove them since #7 already covers this.